docs/use_cases/recommender_systems.md

```diff
@@ -1,15 +1,84 @@
+
+(We deliberately took a detour and observed the problem via graphs. Note that this approach is known as ‘joint matrix factorization’ in the literature).
+
+Once the problem is understood as inherently a graph problem, it is easy to see how additional information can be added. What we want to achieve is to somehow let the algorithm know about the new links, which would help group similar items or users together. In other words, we want to ‘inject’ new nodes, which would link the nodes belonging to the same group. How could this be achieved? For the sake of simplicity, let’s assume that only genre information is available on the items. In this case, for each genre, we add a dummy user whose interactions are items that belong to this genre. In such a way, given a graph of user and item nodes, the new dummy user allows for coupling of same-genre items, which in turns helps the model.
+
+
+#### Illustration
+
+Have a look at the illustration above. There are three users, U1, U2 and U3, and four items, I1, I2, I3, I4. The user U1 has interacted with items I1 and I3. Now, there is a dummy user, UD, who links the items that have the same color (I1, I3, I4). This increases the chances of item I4 to be recommended to user U1.
+
+
+#### Adaptation
+
+As discussed above, we only need to add dummy data: in case of only item side information available, add dummy users; in case of only user side information available, add dummy items; in case both the user and the item information is available, add both the dummy users and the dummy items. Obviously, the dummy nodes should not be part of recommendations; these nodes are only there to help ‘inject’ some ‘commonality’ of the nodes belonging to a certain group.
+
+Beside MF, the same approach can be used with EASE, for example, or with an explicitly graph-based approach for recommendations such as PageRank. In fact, with PageRank, the walks would include the dummy nodes.
+
+One question remains: which weights (ratings) to put for the dummy user interactions? We suggest you first start with low weights, and see how the quality of the recommendations changes.
+
+
+### Potential problems
+
+Obviously, the addition of dummy nodes increases the computational and memory complexity, but in most cases this is a negligible compromise, given the scarcity of the side information. In an audio-on-demand platform, with 250K items in the catalog, there were a few hundred item categories only.
+
+#### Real-world use case
+
+The ALS-based model described above improved upon the production model in an offline evaluation carried on data of one of the most popular audio-on-demand platforms in Germany.
```

Comment:
I would start with this and share the magnitude of the improvement - this is what hooks the user in.. "We did X and gained Y% over an older method Z - here is how:" if you start like this, I keep reading! :)